src/codeu/chat/server/Server.java

```diff
@@ -49,6 +51,8 @@
   private static final Logger.Log LOG = Logger.newLog(Server.class);
 
   private static final int RELAY_REFRESH_MS = 5000;  // 5 seconds
+  //private static final ServerInfo info = new ServerInfo();
+  private static ServerInfo SERVER_INFO;
```

Comment:
this should be final because it is never expected to change during the lifetime of the server